bugwarrior/services/jira.py

```diff
@@ -269,6 +275,9 @@ def get_subtasks(self):
     def get_issue_type(self):
         return self.record['fields']['issuetype']['name']
 
+    def get_assignee(self):
+        return self.record['fields']['assignee']['emailAddress']
```

Comment:
I think we would normally use a username here rather than an email address.

Reply:
I agree, a simple user name wasn't available:

```
 'assignee': {'accountId': '5e18fbf9d52...88579',
              'accountType': 'atlassian',
              'active': True,
              'avatarUrls': {'16x16': 'https://avatar-...-48a3-843a-29184114a6e5/16',},
              'displayName': 'Jake Brinkmann',
              'emailAddress': 'jb...@....com',
              'self': 'https://sonifi.atlassian.net/rest/api/2/user?accountId=5e18fbf9d52...88579',
              'timeZone': 'America/Chicago'},
```

maybe I will make two UDAs, one for email and one for display name. 

the same data structure also exists for `'creator'`, `'reporter'` if I should consider bringing those in as UDAs as well

Reply:
Ah, that's an unfortunate constraint but now I understand your choice.